torch/csrc/autograd/input_buffer.h

```diff
@@ -43,7 +43,6 @@ struct InputBuffer {
   // Returns the inputs as a list of variables. Destroys given InputBuffer.
   static std::vector<Variable> variables(InputBuffer&& g);
 
- private:
```

Comment:
Why?

Reply:
Let me see if I can avoid this.

Reply:
I don't think we can easily avoid this for now. It should be gone for sure though when we eventually remove the usage of captures hooks, so should just be a temporary hack.